src/qtt/instrument_drivers/settings.py

```diff
@@ -0,0 +1,54 @@
+from qcodes import Instrument, ManualParameter
+from qcodes.utils.validators import Numbers
+
+
+class SettingsInstrument(Instrument):
```

Comment:
@ammar92 The SettingsInstrument is not used anywhere in the code. It is also not clear what this is used for. Can it be removed?

Reply:
@peendebak The VirtualAwg requires an instrument that holds the settings/configuration and for this reason and for consistency a `SettingsInstrument` is implemented

Reply:
@ammar92 Then I suggesty renaming the class, documenting it and moving it into the VirtualAwg subdirectory.